tools/text_processing/text_processing/macros.xml

```diff
@@ -6,7 +6,7 @@
         </requirements>
     </xml>
     <token name="@TOOL_VERSION@">9.3</token>
-    <token name="@VERSION_SUFFIX@">1</token>
+    <token name="@VERSION_SUFFIX@">2</token>
```

Comment:
I would bump just the tool. It'll create a new installable revision but at least there won't be bumps for tools that haven't changed.

Reply:
Is that the case? Ok, then we can do it.

Reply:
```suggestion
    <token name="@VERSION_SUFFIX@">1</token>
```